### PR TITLE
User can create an entry

### PR DESCRIPTION
**Pull request by brian-luong**
coding-journal

### CODE REVIEW

File: js/data.js [hunk 1]
```diff
@@ -6,3 +6,43 @@ var data = {
...
+
+  var entry = {
+    title: titleValue,
+    photoUrl: photoUrlValue,
+    notes: notesValue,
+    entryId: data.nextEntryId
+  };
+  data.nextEntryId++;
+  data.entries.unshift(entry);
+});
```
Comment (BrettAlbright) [inline js/data.js]: All of this functionality should be in the `main.js` file. The only thing you want in the data.js file is the `data` object and the local storage functionality.

File: js/main.js [hunk 2]
```diff
@@ -1,2 +1,9 @@
 /* global data */
 /* exported data */
+var $img = document.querySelector('img');
+var $form = document.querySelector('form');
+
+$form.addEventListener('submit', function (event) {
+  $form.reset();
+  $img.src = 'images/placeholder-image-square.jpg';
+});
```
Comment (BrettAlbright) [inline js/main.js]: this should  be in the same callback function as the form data to combine the functionality.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
a few small updates are needed!

File: js/main.js (hunk 2 above)
Comment (brian-luong) [reply js/main.js]: Thanks @Zhiwutian , I've made those changes as you suggested and updated the pull request

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**